packages/mock-app/src/constants/app-config.example.json

```diff
@@ -5,6 +5,74 @@
     "secondaryColor": "white",
     "tertiaryColor": "black"
   },
+  "generateFeatures": [
+    {
+      "name": "General features",
+      "type": "",
+      "styles": {
+        "primaryColor": "rgb(41, 171, 48)",
+        "secondaryColor": "white",
+        "tertiaryColor": "black"
+      },
+      "features": [
+        {
+          "name": "Conformity Credential",
+          "id": "conformity_credential",
+          "components": [
+            {
+              "name": "ConformityCredential",
+              "type": "",
+              "props": {
+                "credentialRequestConfigs": [
+                  {
+                    "url": "http://example.com/deforestation-free-assessment",
+                    "params": {},
+                    "options": {
+                      "headers": [],
+                      "method": "POST"
+                    },
+                    "credentialName": "Deforestation Free Assessment",
+                    "credentialPath": "/body/credentil",
+                    "credentialAppExclusive": "Farm"
+                  },
+                  {
+                    "url": "http://example.com/ce",
+                    "params": {},
+                    "options": {
+                      "headers": [],
+                      "method": "GET"
+                    },
+                    "credentialName": "CE Marking",
+                    "credentialPath": "",
+                    "credentialAppExclusive": "Farm"
+                  },
+                  {
+                    "url": "http://example.com/conformity",
+                    "params": {},
+                    "options": {
+                      "headers": [],
+                      "method": "POST"
+                    },
+                    "credentialName": "Conformity",
+                    "credentialAppExclusive": "Feedlot"
+                  }
+                ],
+                "storedCredentials": {
```

Comment:
`storedCredentials` name is not good. This is a configuration for storage service